csclient/params/helpers.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package params
+
+import (
+	"gopkg.in/juju/charm.v6-unstable/resource"
+)
+
+// Resource2API converts a charm resource into an API Resource struct.
+func Resource2API(res resource.Resource) Resource {
```

Comment:
It'd be nice is a resource.Resource was just marshalable without the need for conversion. I assume there are solid reasons why it isn't though.